tests/operator_test.go

```diff
@@ -410,6 +429,34 @@ var _ = Describe("Operator", func() {
 			sanityCheckDeploymentsExist()
 		})
 
+		It("should be able to delete and re-create kubevirt install in a different namespace", func() {
+			allPodsAreReady(tests.KubeVirtVersionTag)
+			sanityCheckDeploymentsExist()
+
+			By("Deleting KubeVirt object")
+			deleteAllKvAndWait(false)
+
+			// this is just verifying some common known components do in fact get deleted.
+			By("Sanity Checking Deployments infrastructure is deleted")
+			sanityCheckDeploymentsDeleted()
+
+			By("Creating KubeVirt Object")
+			newKV := copyOriginalKv()
+			newKV.Name = "kubevirt-default-namespace"
+			newKV.Namespace = "default"
```

Comment:
Hm, can we create an extra namespace for this? That we are still cleaning the "default" namespace is actually a relikt. I would prefer a specific "kubevirt-install-alternative" namespace, which we then also clean.

Reply:
i'd be okay with this if we remove "default" entirely from the cleanup list. 

Is there a reason we still have "default" in that list? I'm not aware of one.

Reply:
Removing sounds perfect. The intention was to keep the cleanup for a few weeks, but now it is probably a year already.

Reply:
i decided to go with the kubevirt-test-default namespace. That namespace is isolated to the test-suite and cleaned up by the test-suite. It prevents us from having to pre-define another namespace before running the test suite.